recipe/meta.yaml

```diff
@@ -280,19 +294,31 @@ outputs:
         -     python setup.py install -v -v
         -     python -c "import foobar"
         -   popd
-        -   pushd prefix-replacement  # [unix]
-        -     bash build-and-test.sh  # [unix]
+        -   pushd embedding-interpreter  # [unix]
+        -     bash build-and-test.sh  # [unix and not (linux and aarch64)]
         -   popd  # [unix]
-        -   pushd cmake
-        -     cmake -GNinja -DPY_VER={{ version }}
-              # --trace --debug-output --debug-trycompile .
+        -   pushd cmake  # [not (linux and ppc64le)]
```

Comment:
Why?

Reply:
I'm not exactly sure what you are asking here @isuruf.

This test was originally going to about prefix-replacement but it is not about that anymore so it's been renamed. Now it's a very comprehensive test to make sure we can embed both static and shared python interpreters in other programs.

Reply:
The cmake test is disabled for aarch64 atm because our cmake is too old there I think @katietz ?

Reply:
Yes, sorry. I meant to ask why it's skipped for `ppc64le`?